packages/perseus/src/widgets/definition/definition.tsx

```diff
@@ -64,18 +65,29 @@ class Definition extends React.Component<DefinitionProps> implements Widget {
             <DefinitionConsumer>
                 {({activeDefinitionId, setActiveDefinitionId}) => (
                     <Popover
+                        dismissEnabled
                         content={
-                            <PopoverContentCore
-                                style={styles.tooltipBody}
-                                closeButtonVisible={true}
+                            <View
+                                onKeyDown={(e) => {
+                                    // Close popover when tabbing off the close button
+                                    // (the only tabbable element in the popover)
```

Comment:
I don't think we have anything that enforces that no other clickable elements appear in the popover, but the style guide for content creators says to only use basic text, so this should be safe within Khan Academy. I wonder if we need to do something more for other users.